vnet/udpproxy.go

```diff
@@ -93,13 +114,22 @@ func (v *aUDPProxyWorker) Proxy(client *Net, serverAddr *net.UDPAddr) error { //
 		return err
 	}
 
+	// Create context for cleanup.
+	v.ctxDispose, v.ctxDisposeCancel = context.WithCancel(context.Background())
```

Comment:
`v.ctxDispose` seems used only in this scope, so it can be a local variable.

Reply:
It's also used in Close(), which will call `v.ctxDisposeCancel()` to cancel the `v.ctxDispose`.

Reply:
Only `CancelFunc` is used in `Close()`.

Reply:
OK, I removed the `ctxDispose` from struct.